music21/musicxml/xmlToM21.py

```diff
@@ -4666,6 +4666,14 @@ def xmlToRepeat(self, mxBarline, inputM21=None):
         'final'
         >>> r.direction
         'end'
+
+        Test that a forward repeat with times doesn't raise an exception, and
+        that the resulting Repeat doesn't have times set.
+        >>> mxStartBarline = ET.fromstring('<barline><bar-style>light-heavy</bar-style>' +
```

Comment:
A gotcha for future: if you build the documentation with `documentation/make.py` you'll notice this doesn't render as code without a blank line before the code block:
```suggestion

        >>> mxStartBarline = ET.fromstring('<barline><bar-style>light-heavy</bar-style>' +
```